Example/PageBuilderCountdown/view/base/web/js/content-type/example-countdown/appearance/default/widget.js

```diff
@@ -0,0 +1,51 @@
+define([
+    'jquery'
+], function ($) {
+    'use strict';
+
+    return function (config, element) {
+        var countdownField = element[0].children[0],
+            intervalId = 0;
+
+        intervalId = setInterval(function (field, intervalId) {
+            let target = '';
+            let countdownField = field[0].children[0];
+
+            if (countdownField) {
+                target = countdownField.getAttribute('target_date');
+            }
+
+            if (!target || target === '') {
```

Comment:
You have this logic duplicated across the admin and the store front, you could write a function within base and import it into both areas to save on duplicating code.